validphys2/src/validphys/process_options.py

```diff
@@ -157,39 +155,50 @@ def _dijets_xq2map(kin_info):
 
 def _hqp_yq_xq2map(kin_info):
     # Compute x, Q2
-    if {"k1", "k2", "k3"} <= kin_info.keys():
-        kin_info[_Vars.y_t] = kin_info["k1"]
-        kin_info[_Vars.m_t2] = kin_info["k2"]
-        kin_info[_Vars.sqrts] = kin_info["k3"]
-
-    mass2 = kin_info.get_one_of(_Vars.m_t2, _Vars.m_ttBar)
-
+    # Theory predictions computed with HT/4 see 1906.06535
+    mass2 = kin_info[_Vars.m_t2]
     ratio = np.sqrt(mass2) / kin_info[_Vars.sqrts]
     x1 = ratio * np.exp(kin_info[_Vars.y_t])
     x2 = ratio * np.exp(-kin_info[_Vars.y_t])
     q2 = mass2
     x = np.concatenate((x1, x2))
-    return np.clip(x, a_min=None, a_max=1, out=x), np.concatenate((q2, q2))
+    return np.clip(x, a_min=None, a_max=1, out=x), np.concatenate((q2, q2)) / 4
 
 
 def _hqp_yqq_xq2map(kin_info):
     # Compute x, Q2
-    mass2 = kin_info.get_one_of(_Vars.m_t2, _Vars.m_ttBar)
+    # Theory predictions computed with HT/4 see 1906.06535
+    mass2 = kin_info[_Vars.m_t2]
     ratio = np.sqrt(mass2) / kin_info[_Vars.sqrts]
     x1 = ratio * np.exp(kin_info[_Vars.y_ttBar])
     x2 = ratio * np.exp(-kin_info[_Vars.y_ttBar])
     q2 = kin_info[_Vars.m_t2]
     x = np.concatenate((x1, x2))
-    return np.clip(x, a_min=None, a_max=1, out=x), np.concatenate((q2, q2))
+    return np.clip(x, a_min=None, a_max=1, out=x), np.concatenate((q2, q2)) / 4
 
 
 def _hqp_ptq_xq2map(kin_info):
     # Compute x, Q2
-    QMASS2 = TMASS * TMASS
-    Q = np.sqrt(QMASS2 + kin_info[_Vars.pT_t] * kin_info[_Vars.pT_t]) + kin_info[_Vars.pT_t]
+    Q = (kin_info[_Vars.m_t2] + kin_info[_Vars.pT_t] * kin_info[_Vars.pT_t]) ** 0.5 / 2
+    return Q / kin_info[_Vars.sqrts], Q * Q
+
+
+def _hqp_mqq_xq2map(kin_info):
+    # Compute x, Q2
+    Q = kin_info[_Vars.m_ttBar] / 4
     return Q / kin_info[_Vars.sqrts], Q * Q
 
 
+def _inc_xq2map(kin_info):
+    # Compute x, Q2
+    if {"k1", "k2", "k3"} <= kin_info.keys():
+        mass2 = kin_info["k2"]
+    else:
+        mass2 = kin_info.get_one_of("m_W2", "m_Z2", "m_t2")
```

Comment:
```suggestion
    # k2 necessary to take the mass for DY inclusive cross sections still not migrated
    mass2 = kin_info.get_one_of(_Vars.m_W2, _Vars.m_Z2, _Vars.m_t2, "k2")
```

Just to make the suggestion more explicit.

Reply:
yes yes it was clear, I just hadn't gotten to correcting it yet xD